pages/common/lua.md

```diff
@@ -0,0 +1,15 @@
+# lua
+
+> A powerful, light-weight embedded programming language.
+
+- Start an interactive lua shell:
+
+`lua`
+
+- Execute a lua script:
+
+`lua {{script_name.lua}} {{--optional-argument}}`
+
+- Execute a string of lua:
+
+`lua -e '{{insert lua code here}}`
```

Comment:
Yes, a concrete example would be preferable any day. 

Sth like - `lua -e '{{print("Hello World")}}'`